src/distribution-integration/alpm-hook.sh

```diff
@@ -2,12 +2,9 @@
 
 set -euo pipefail
 
-# enumerate files that are managed by holo-files
-holo scan --short | grep '^file:' | cut -d: -f2- | sort > /tmp/files-managed-by-holo
-
-# changed files come from stdin (but without leading slash!); find files
-# managed by Holo among these
-sort | sed 's+^+/+' | comm -12 - /tmp/files-managed-by-holo | sed 's/^/file:/' | xargs -r holo apply
-
-# cleanup
-rm /tmp/files-managed-by-holo
+comm -12 \
+     <(# changed files come from stdin (but without leading slash!)
+       sort | sed 's+^+/+') \
+     <(# enumerate files that are managed by holo-files
+       holo scan --short | sed -n 's/^file://p') \
+| sed 's/^/file:/' | xargs -r holo apply
```

Comment:
I did.  But now that you say that, I'm figuring out how to add it to the test suite.

In doing so, I realized that it doesn't work correctly with HOLO_ROOT_DIR.  I guess I'll fix that.  But it's being highlighted to me that holo-files embeds HOLO_ROOT_DIR into the entity IDs, which seems gross to me.